src/store/accounts/nonce.ts

```diff
@@ -20,11 +22,27 @@ export const latestNonceState = atom(get => {
   return get(localNoncesState([network.url, address || '']));
 });
 
+// TODO use AccountsApi.getAccountNonces instead (not yet available in @stacks/blockchain-api-client)
+export const lastApiNonce = atom(async get => {
+  const network = get(currentNetworkState);
+  const address = get(currentAccountStxAddressState);
+  if (!address) return;
+  try {
+    return fetchAccountNonce(network.url)(address);
+  } catch (err) {
+    console.log('Not able to fetch nonce from api');
+    return
```

Comment:
This should be a prettier error

Reply:
This error is for debugging purpose not for the user since we fallback to our current nonce calculation if the api is down. What other info would you like to see here ?

Reply:
I'm just saying that prettier should add a `;` here. WIthout it, it should be a prettier check CI error?

Reply:
Oh ok I get it. Maybe this is the prettier config. All checks are passing locally. I will check again later.